app/middlewares/user.js

```diff
@@ -0,0 +1,36 @@
+const { checkSchema, validationResult } = require('express-validator');
+
+exports.dataUser = (req, res, next) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(422).json({ errors: errors.mapped() });
```

Comment:
Remember that we return errors using custom errors, here you should create one and pass it to the `next` function instead of returning the `res`.